### PR TITLE
add main.c

### PR DESCRIPTION
**Pull request by in-siders17**

### CODE REVIEW

File: KP2/CSolution/main.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    scanf("%f", &x);
+
+    printf("Enter y value:");
+    scanf("%f", &y);
+
+    if (x > 0, x * y > 1) f = (x + y) / (1 - x * y);
+    else f = (pow(x, 2) - y) / (1 - x * y);
+
+    printf("The function is equal to = %f", f);
+    getch();
```
Comment (tenk28) [inline KP2/CSolution/main.c:19]: For windows only, remove pls

File: KP2/CSolution/main.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <math.h>
+
+
+int main() {
+    
+    double y, x, f;
+
+    printf("Enter x value: ");
+    scanf("%f", &x);
```
Comment (tenk28) [inline KP2/CSolution/main.c:10]: type of 'x' is double,
please use "%lf" specifier

`scanf("%lf", &x);`

File: KP2/CSolution/main.c [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    printf("Enter y value:");
+    scanf("%f", &y);
```
Comment (tenk28) [inline KP2/CSolution/main.c:13]: same correct to "%lf" specifier

File: KP2/CSolution/main.c [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    if (x > 0, x * y > 1) f = (x + y) / (1 - x * y);
```
Comment (tenk28) [inline KP2/CSolution/main.c:15]: make better format:
if (x > 0, x * y > 1)
    f = (x + y) / (1 - x * y);
else
    f = (pow(x, 2) - y) / (1 - x * y);

refer to:
https://github.com/MaJerle/c-code-style#compound-statements

Comment (tenk28) [inline KP2/CSolution/main.c:15]: make sure condition is correct:
x > 0, x * y > 1

',' - is not AND statement,
AND statement in C/C++ is `&&`

So correct statement is:
(x > 0 && x * y > 1)

**Review by tenk28 — CHANGES_REQUESTED** [review]

File: KP2/SashaSolution/CSolution/main.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+
+int main() {
+    
+    double y, x, f;
+
+    printf("Enter x value: ");
+    scanf("%lf", &x);
+
+    printf("Enter y value:");
```
Comment (tenk28) [inline KP2/SashaSolution/CSolution/main.c:12]: add space after :

`printf("Enter y value: ");`

File: KP2/SashaSolution/CSolution/main.c [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    scanf("%lf", &y);
+
+    if (x > 0 && x * y > 1) 
+        f = (x + y) / (1 - x * y);
+    else 
+        f = (pow(x, 2) - y) / (1 - x * y);
+
+    printf("The function is equal to = %lf", f);
```
Comment (tenk28) [inline KP2/SashaSolution/CSolution/main.c:20]: add new line symbol at the end 

`printf("The function is equal to = %lf", f);`

File: KP2/SashaSolution/CSolution/main.c [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+    printf("Enter x value: ");
+    scanf("%lf", &x);
+
+    printf("Enter y value:");
+    scanf("%lf", &y);
+
+    if (x > 0 && x * y > 1) 
+        f = (x + y) / (1 - x * y);
+    else
```
Comment (tenk28) [inline KP2/SashaSolution/CSolution/main.c:17]: we need to make additional condition if x == 0 and everything else is could be calculated

Comment (tenk28) [reply KP2/SashaSolution/CSolution/main.c:17]: refer to: https://github.com/tenk28/SashaLearning/blob/b17184eb6a7c98b14870d5b77b3034361997f64d/KP2/DmytroSolution/CSolution/EasySolution/main.c#L9

**Review by tenk28 — CHANGES_REQUESTED** [review]

File: KP2/SashaSolution/CSolution/main.c (hunk 5 above)
Comment (in-siders17) [reply KP2/SashaSolution/CSolution/main.c]: done

File: KP2/SashaSolution/CSolution/main.c (hunk 6 above)
Comment (in-siders17) [reply KP2/SashaSolution/CSolution/main.c]: done

File: KP2/SashaSolution/CSolution/main.c (hunk 7 above)
Comment (in-siders17) [reply KP2/SashaSolution/CSolution/main.c]: done

**Review by tenk28 — APPROVED** [review]

**Pull request merged**